pkg/build/strategies/sti/sti.go

```diff
@@ -575,7 +575,7 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
 				// we're ignoring ErrClosedPipe, as this is information
 				// the docker container ended streaming logs
 				if glog.Is(2) && err != io.ErrClosedPipe && err != io.EOF {
-					glog.Errorf("Error reading docker stdout, %v", err)
+					glog.Errorf("Error reading docker stdout, %#+v", err)
```

Comment:
# does print the type ... I just didn't want to leave any stone un-turned
(in hindsight my experiment was on a complex object and the delta was hard
to discern
between %#v and %#+v)...I'll remove the +

On Thu, Aug 11, 2016 at 2:38 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/build/strategies/sti/sti.go
> <https://github.com/openshift/source-to-image/pull/569#discussion_r74478841>
> :
>
> > @@ -575,7 +575,7 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
> >  				// we're ignoring ErrClosedPipe, as this is information
> >  				// the docker container ended streaming logs
> >  				if glog.Is(2) && err != io.ErrClosedPipe && err != io.EOF {
> > -					glog.Errorf("Error reading docker stdout, %v", err)
> > +					glog.Errorf("Error reading docker stdout, %#+v", err)
>
> just # should be sufficient, no + needed, right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/source-to-image/pull/569/files/2bc64350ff8056db14f16455bee0f5c013391a01#r74478841>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADbadBwtk3RPsnNMZGn3AN6ukTsB0-qNks5qe2w8gaJpZM4Jiann>
> .
>